sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -974,12 +975,12 @@ def test_queue_by_queue_client_conn_str_receive_handler_with_autolockrenew(self,
                     if not messages:
                         messages.append(message)
                         assert not message._lock_expired
-                        renewer.register(receiver, message, max_lock_renewal_duration=10)
+                        renewer.register(receiver, message, max_lock_renewal_duration=20)
```

Comment:
I'm curious why we needed to bump all the durations in the test - it wouldn't work with the existing values?

Reply:
I revert the values, we should be good :)